homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
```

Comment:
For the cloud one, do we always need to fetch the user and all their devices? Or can we slow that down eventually? Not a blocker, but I would be interested to know